charts/kube-prometheus-stack/values.yaml

```
@@ -2890,8 +2890,16 @@ prometheusOperator:
       # self-signed root certificate
       rootCert:
         duration: ""  # default to be 5y
+        # -- Set the revisionHistoryLimit on the Certificate. See
+        # https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1.CertificateSpec
+        # Defaults to nil.
+        revisionHistoryLimit: ""
```

Any downside to define the default value from helm point of view also nil?
```suggestion
        revisionHistoryLimit: 
```

---

Shouldn't be a problem, handled